EqualityInformationApi/V1/Controllers/GetEqualityInformationApiController.cs

```diff
@@ -20,18 +20,10 @@ namespace EqualityInformationApi.V1.Controllers
     public class GetEqualityInformationApiController : BaseController
```

Comment:
Can we not have both the controllers in one class like we do in the other APIs?

Reply:
I thought that was weird too